interfaces/bots/interface_bot.py

```diff
@@ -196,32 +196,32 @@ def get_command_portfolio():
     @staticmethod
     def get_command_profitability():
         has_real_trader, has_simulated_trader, \
-        real_global_profitability, simulated_global_profitability, \
-        real_percent_profitability, simulated_percent_profitability, \
-        real_no_trade_profitability, simulated_no_trade_profitability, \
-        market_average_profitability = get_global_profitability()
+         real_global_profitability, simulated_global_profitability, \
+         real_percent_profitability, simulated_percent_profitability, \
+         real_no_trade_profitability, simulated_no_trade_profitability, \
+         market_average_profitability = get_global_profitability()
         profitability_string = ""
         if has_real_trader:
             profitability_string = "{0}Global profitability : {1} ({2}%), market: {3}%, initial portfolio: {4}%{5}" \
                 .format(
-                REAL_TRADER_STR,
-                PrettyPrinter.portfolio_profitability_pretty_print(real_global_profitability,
-                                                                   None,
-                                                                   get_reference_market()),
-                PrettyPrinter.get_min_string_from_number(real_percent_profitability, 2),
-                PrettyPrinter.get_min_string_from_number(market_average_profitability, 2),
-                PrettyPrinter.get_min_string_from_number(real_no_trade_profitability, 2),
-                EOL)
+                    REAL_TRADER_STR,
```

Comment:
Why keeping `format` here ? and not using f-string

Reply:
right